spec/scenario/Flywheel/Flywheel.scen

```diff
@@ -219,3 +226,38 @@ Test "Comp is not claimed automatically"
     -- Confirm there was comp to claim
     Comptroller ClaimComp Geoff
     Assert Equal (Erc20 COMP TokenBalance Geoff) 200000
+
+Test "New COMP speeds apply to both prior borrowers+suppliers and later borrowers+suppliers correctly"
+    -- Supplying
+    --   Torrey mints 100e18 before COMP speeds set
+    --   Jared mints 100e18 after COMP speeds set
+    -- Borrowing
+    --   Coburn borrows 10e18 before COMP speeds set
+    --   Geoff borrows 10e18 after COMP speeds set
+    FlywheelComptroller
```

Comment:
this seems like a good test, but just b/c of the name of the commit I'll note that this isn't what my test was, since this one starts on the new comptroller. The bug was on transitioning between the old and new implementation